usr/bin/gg

```diff
@@ -32,12 +32,12 @@ FILTER=false
 EXPERIMENTAL=false
 
 function usage() {
-	echo "Usage: gg [OPTION]... [FILE]"
-	echo "-i, --ignore, -f, --filter 			Enable filtering"
+	echo "Usage: gg [OPTION]... <FILE>"
+	echo "-i, --ignore, -f, --filter     (default: false) Enable filtering"
 	echo "-v, --version 					Display version"
 	echo "-h, --help 					Show this list"
-	echo "-d, --disable ["${products[@]}"]			disable product filter"
-	echo "-e, --experimental				enable experimental features"
+	echo "-d, --disable ["${products[@]}"](default: none)  exclude product patterns"
+	echo "-e, --experimental	       (default: false) enable experimental features"
```

Comment:
First word of description should be capitalized to match other lines.
'Exclude' and 'Enable'

Reply:
oops....  Miissed that.